walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -47,16 +47,36 @@
     }
 
     // Add optional guests
-    // TODO: Use stream to make a deep copy
-    List<TimeRange> optionalMeetingTimes = new ArrayList<TimeRange>();
-    for (int i = 0; i < meetingTimes.size(); i++) {
-      optionalMeetingTimes.add(meetingTimes.get(i).fromStartDuration(meetingTimes.get(i).start(), 
-          meetingTimes.get(i).duration()));
-    }
-    optionalMeetingTimes = findSchedule(possibleTimes, events, optionalGuests, request);
-    if (!optionalMeetingTimes.isEmpty() || attendees.isEmpty()) {
+    List<TimeRange> optionalMeetingTimes = copyListTimeRnage(meetingTimes);
+    optionalMeetingTimes = findSchedule(timeRangeToRangeSet(meetingTimes), events, optionalGuests, request);
```

Comment:
Looks like the work in line 50 is thrown away in line 51. Maybe you just need line 51?